packages/belos/tpetra/test/Native/nonsymm_matrix.cpp

```diff
@@ -232,12 +235,15 @@ testSolver (Teuchos::FancyOStream& out,
   params->set ("Verbosity", verbose ? 1 : 0);
   params->set ("Maximum Iterations", commandLineOptions.maxNumIters);
   params->set ("Num Blocks", commandLineOptions.restartLength);
+  params->set ("Orthogonalization", commandLineOptions.orthoType);
   if (solverName == "TPETRA GMRES S-STEP") {
     params->set ("Step Size", commandLineOptions.stepSize);
-    params->set ("Compute Ritz Values", commandLineOptions.computeRitzValues);
```

Comment:
Why did we remove this option?

Reply:
I don't think we removed it. We just moved it to [here](https://github.com/trilinos/Trilinos/blob/c27e31e8397f5bdf4d7861be118f01e23f522faf/packages/belos/tpetra/test/Native/nonsymm_matrix.cpp#L245).